src/transactions/test/ClawbackTests.cpp

```diff
@@ -112,7 +120,12 @@ TEST_CASE("clawback", "[tx][clawback]")
                     gateway.allowTrust(idr, a1, 0);
                 }
             }
-
+            SECTION("clawback when issuer already has INT64_MAX liabilities")
```

Comment:
Nitpick: some whitespace would be appreciated between unrelated sections.

Reply:
Done